README.md

```diff
@@ -1,2 +1,72 @@
-# faraday_connection_pool
+# FaradayConnectionPool
+
 FaradayConnectionPool provides a persistent Net::HTTP Faraday adapter.
+Unlike Net::Persistent, connections are pooled across threads and fibres so that they can be reused more frequently.
```

Comment:
Net::HTTP::Persistent

I think I would say:

"Unlike Net::HTTP::Persistent, which has a connection-per-thread, connections are pooled across all threads and you will always get the most recently used connection. This should mean that you are more likely to get an existing connection with a reduced chance of getting a connection reset."